app/src/main/res/values/strings.xml

```diff
@@ -770,4 +770,7 @@
     <string name="title_descriptions_onboarding_got_it">Got it</string>
     <string name="translate_descriptions_onboarding_got_it">Got it</string>
     <string name="android_app_edit_help_url">https://m.mediawiki.org/wiki/Wikimedia_Apps/Suggested_edits</string>
+    <string name="suggested_edits">Suggested edits</string>
+    <string name="feed_item_type_suggested_edit">Suggestions based on your language and editing history.</string>
```

Comment:
Can please put the `feed_item_type` strings under the `feed_item_type_randomizer` string? It would be very helpful for future maintenance.

Reply:
Done